unit-tests/py/rspy/devices.py

```diff
@@ -74,7 +74,11 @@ def __init__( self, sn, dev ):
 
         self._usb_location = None
         try:
-            self._usb_location = _get_usb_location(self._physical_port)
+            if dev.supports(rs.camera_info.connection_type):
+                self._connection_type = dev.get_info(rs.camera_info.connection_type)
+                self._is_dds = self._connection_type == "DDS"
+                if self._connection_type == "USB":
```

Comment:
Not really needed any more, as `_get_usb_location` should now return `None` for DDS devices after my PR